README.md

```diff
@@ -25,165 +28,182 @@ Check the [roadmap here](https://github.com/Kureev/react-native-blur/issues/1)
   ```
   react-native link react-native-blur
   ```
-3. (Android only) Add the following to your `android/app/build.gradle`
-
-`android/build.gradle`
-```
-  buildscript {
-    dependencies {
-        // Update "Android Plugin for Gradle" version
-        classpath 'com.android.tools.build:gradle:2.2.3'
-    }
-  }
 
-  // ...
+3. (Android only) Add the following to `android/app/build.gradle`
 
-  allprojects {
-    repositories {
-        // Add the following in a new line, underneath "maven { url '$rootDir/../node_modules/react-native/android' }"
-        maven { url 'https://github.com/500px/500px-android-blur/raw/master/releases/' }
-    }
-}
 ```
-
-`android/app/build.gradle`
-```
-dependencies {
-    compile 'com.fivehundredpx:blurringview:1.0.0'
-}
-
-
 android {
+    // ...
     defaultConfig {
-        // Add these to the existing config
+        // Add these lines below the existing config
         renderscriptTargetApi 23
         renderscriptSupportModeEnabled true
     }
 }
 ```
 
-`android/gradle/wrapper/gradle-wrapper.properties`
-```
-// Update Gradle version
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
-```
-
-4. Inside your code include JS part by adding
+4. Include the library in your code:
 
   ```javascript
+  import { BlurView, VibrancyView } from 'react-native-blur';
+
+  // OR
+
   const { BlurView, VibrancyView } = require('react-native-blur');
   ```
 
 5. Compile and have fun!
 
-### Usage example
-
-You can run the built-in examples by running these steps:
-
-
-1. Clone the repository
-
-```
-cd ~
-git clone https://github.com/react-native-community/react-native-blur.git
-```
-
-2. cd to `examples/Basic`
-
-```
-cd react-native-blur/examples/Basic
-```
-
-3. Install dependencies
-
-```
-npm install
-```
-
-4. Run the apps:
-
-#### Run the iOS app
 
-```
-react-native run-ios
-```
+### BlurView
 
-#### Run the Android app
+**Properties:**
 
-```
-react-native run-android
-```
+- `blurType` (String)
+  - `xlight` - extra light blur type
+  - `light` - light blur type
+  - `dark` - dark blur type
+- `blurAmount` (Default: 10, Number)
+  - `0-100` - Adjusts blur intensity
```

Comment:
Oh wow, I hadn't seen that before. I've just checked on iOS, and `blurAmount` can go beyond 100, but on Android it crashes if it goes above 25. I think it would make sense to clamp the value to `0 < radius <= 25` on Android, but allow it to be higher on iOS. I'll change that and add a note to the README. I think I'll also show a warning if you try to explicitly set a `blurRadius` higher than 25.